### PR TITLE
Add AuthTests class

### PR DESCRIPTION
**Pull request by YummyMirror**

### CODE REVIEW

File: src/test/java/com/practice/git/test/AuthTests.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+/**
+ * @author anatoli.anukevich on 20:48 29-Mar-21
+ */
+public class AuthTests {
+    private WebDriver driver;
+
+    @BeforeClass(alwaysRun = true)
+    public void setUp() {
+        this.driver = new ChromeDriver();
+    }
```
Comment (YU-alt-acc) [inline src/test/java/com/practice/git/test/AuthTests.java:21]: Why did you not create a BaseTest class and put fixtures into it?

File: src/test/java/com/practice/git/test/AuthTests.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        navigator.mainPage();
+
+        LoginPageObject login = new LoginPageObject(this.driver);
+        login.loginAs(new User().setUsername("").setPassword(""));
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void tearDown() {
+        this.driver.quit();
+    }
```
Comment (YU-alt-acc) [inline src/test/java/com/practice/git/test/AuthTests.java:35]: The same as with BeforeClass

File: src/test/java/com/practice/git/test/AuthTests.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    private WebDriver driver;
+
+    @BeforeClass(alwaysRun = true)
+    public void setUp() {
+        this.driver = new ChromeDriver();
+    }
+
+    @Test
+    public void validLoginTest() {
+        Navigator navigator = new Navigator(this.driver);
```
Comment (YU-alt-acc) [inline src/test/java/com/practice/git/test/AuthTests.java:25]: No lazy initialization

**Review by YU-alt-acc — CHANGES_REQUESTED** [review]

File: src/test/java/com/practice/git/application/ApplicationManager.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        }
+        return this.navigator;
+    }
+
+    public LoginPageObject login() {
+        if (Objects.isNull(this.login)) {
+            this.login = new LoginPageObject(this.driver);
+        }
+        return this.login;
+    }
```
Comment (YU-alt-acc) [inline src/test/java/com/practice/git/application/ApplicationManager.java:42]: Why are they not thread safety?

**Review by YU-alt-acc — CHANGES_REQUESTED** [review]

File: src/test/java/com/practice/git/application/ApplicationManager.java (hunk 4 above)
Comment (YummyMirror) [reply src/test/java/com/practice/git/application/ApplicationManager.java:42]: It will be added later

**Pull request merged**